java/configuration/src/main/java/sleeper/configuration/properties/table/TableProperty.java

```diff
@@ -83,6 +83,15 @@ public interface TableProperty extends SleeperProperty {
             .description("A unique ID identifying this table, generated by Sleeper on table creation.")
             .propertyGroup(TablePropertyGroup.DATA_DEFINITION)
             .setBySleeper(true).build();
+    TableProperty TABLE_ONLINE = Index.propertyBuilder("sleeper.table.online")
+            .description("A boolean flag representing whether this table is online or not.\n" +
+                    "An offline table will not have any partition splitting or compaction jobs run automatically.\n" +
+                    "Note that taking a table offline will not stop any partitions that are being split or compaction " +
+                    "jobs that are running.")
```

Comment:
Add note that data can still be ingested to an offline table, and queries can still run against an offline table.